src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
-import styles from './GameCardImage.module.css';
+
 import notLiked from '../../../../assets/GameCard/like/not-liked.svg';
```

Comment:
You can use the assets alias

Reply:
yes! but thats not my component. Its Vales

Reply:
Oh, thought you changed that, my bad!